### PR TITLE
User registration (Sign up page)

### PR DESCRIPTION
**Pull request by CATcommander**
Fixed the user registration issues. Proper error messages are displayed upon wrong credentials or empty login. Please do a code review and let me know if you see any errors. I will migrate first and last name because currently we only have name field.

### CODE REVIEW

File: app/views/sessions/new.html.erb [hunk 1]
```diff
@@ -1,26 +1,27 @@
...
-    <%= formElement.label :Username %>
-    <%= "/" %>
-    <%= formElement.label :email%>
-    <%= formElement.email_field :email,  class: 'form-control', id: 'Username', placeholder: 'radventureit@gmail.com' %>
-  </div>
+      <div>
+        <%= formElement.label :password%>
+        <%= formElement.password_field :password, class: 'form-control', id: 'Password', placeholder: 'Password' %>
+      </div>
+      </br>
```
Comment (lewisric) [inline app/views/sessions/new.html.erb]: Is this break necessary?

**Comment by lewisric** [comment]
Im ok with the code need another pair of eyes however

**Comment by lewisric** [comment]
@beazley2 @ardeshirj @ankur-gupta91 @sector46 
Please take a glance so we can merge

**Comment by beazley2** [comment]
I'm ok with it

**Comment by CATcommander** [comment]
If you guys look at line 29 and 39 in  app/views/users/new.html.erb, they both point to instance variable *:name*. It would make a lot more sense, given the DB schema and how much time we have, if we changed first and last name to just name. That way, we will be asking user for their full name not first and last (like now).

**Comment by ardeshirj** [comment]
The heroku commits accidentally got merged to this PR branch, but I fixed it in master.

**Pull request merged**